test/integration/mongodb-handshake/promise_handshake.test.js

```diff
@@ -28,7 +28,7 @@ describe('Handshake', function () {
         // Execute legacy hello command
         client
           .db(configuration.db)
-          .command({ [LEGACY_HELLO_COMMAND]: true })
+          .command({ [LEGACY_HELLO_COMMAND]: 1 })
```

Comment:
This test file doesn't add much value to our suite, + the strange constraints like poolsize, using legacy format strings, only on single, promise usage combined with mocha done style. Can we just delete this? 😅

Reply:
Removed.